pkg/markdown/markdown.go

```diff
@@ -92,6 +94,19 @@ func GetStyle(defaultStyle string) string {
 	return "notty"
 }
 
+func formatImgTags(content string) string {
+	lines := strings.Split(content, "\n")
+	re := regexp.MustCompile(`<img[^>]+\bsrc=["']([^"']+)["']`)
```

Comment:
This part is getting tricky because you're basically doing HTML parsing without writing a full-fledged HTML parser. Note that attribute values such as `src=VALUE` don't necessarily have to be closed in quotes. Furthermore, images can have text in the `alt` attribute, which would be important to parse out and preserve. Finally, an `<img>` tag can be surrounded by regular content (like `hello <img src="..."> world!`), so we can't just assume that an `<img>` tag will always be on its own line and that the rest of the content can be discarded. Let's only replace the `<img...>` portion within the original content with the `![](...)` syntax and preserve everything else.

Reply:
Additionally this doesn't handle image tags in code blocks which was the issue with our initial emoji rendering solution. Overall, I think we should be adding this feature into the upstream `glamour` library instead of here. @mislav what do you think?

Reply:
Oh yeah 🤦 excellent point @samcoe. I do agree that this should be handled at the level that parses Markdown (so, in Glamour) rather than with regexp matching.

Reply:
It sounds like the glamour team is uninterested in handling HTML: https://github.com/charmbracelet/glamour/issues/111#issuecomment-811291959

If that is the case, we may want to consider an alternative to glamour in the long run.

I'm going to mark this PR as a draft so we can continue discussion.